docs/source/config_options.rst

```diff
@@ -1,6 +1,12 @@
-*****************
+.. _config options:
+
+=============
+Configuration
+=============
+
+-----------------
 Config Parameters
-*****************
+-----------------
```

Comment:
I don't see the realtime address in this list, I'm assuming it's supposed to still be in here?

Reply:
There were other config parameters missing too, updated them here: c6a4ef697ba048eae82697c3a769474429b0a59f